src/bindings/python/src/openvino/utils.py

```diff
@@ -38,6 +41,31 @@ def _add_openvino_libs_to_search_path() -> None:
                 os.add_dll_directory(os.path.abspath(lib_path))
 
 
+def get_cmake_path():
+    """Searches for the directory containing CMake files within the package install directory.
+
+    Parameters:
+    None
+
+    Returns:
+    str: The path to the directory containing CMake files, if found. Otherwise, returns empty string.
+    """
+    site_packages = chain(
+        (Path(__file__).parent.parent,),
+        site.getusersitepackages(),
+        site.getsitepackages()
+    )
+
+    cmake_paths = [
+        site_package / __package__ / "cmake"
+        for site_package in map(Path, site_packages)
+    ]
+
+    cmake_path = next((path for path in cmake_paths if path.is_dir()), "")
```

Comment:
should we check if `path / OpenVINOConfig.cmake` exists?

Reply:
solved